docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,381 @@
+
+Communication between MSPKI and PacketFence will be using port 80 with this configuration.
+
+Configuring Network Device Enrollement Service (NDES)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+NOTE: In this guide you will see a lot of use of <ServerDNSName>, most of the MSPKI services requires in their configuration to use the FQDN of the server and not his IP.
+
+Our main activity in the deployment of ADCS is to configure Network Enrollement Device Service (NDES) which will allow us to exchange certificate with the MS PKI server via Simple Certificate Exchange Protocol (SCEP).
+
+Every configuration change has to be done by an account with administrative privileges.
+
+To allow the use of NDES as a Pki-Provider in BOYD deployement there is multiple changes.
+
+Challenge Password
+^^^^^^^^^^^^^^^^^^
+
+MSCEP include by default a challenge password, which is unique and dynamically generated for each device which want to enroll on the system. The issue here while using the challenge password is that it is necessary for an administrator to connect to the NDES server from the user's device. This a barrier to BOYD deployement for instance, where you want the user to be able to register himself his device. Since we use NDES with PacketFence, our security to obtain a certificate would be the credentials necessary to access the enrollement system.
+
+To disable the challenge password you need to modifiy the following key in the Windows registry. 
+
+Click `Start` and enter `regedit`.
+
+Navigate to `Computer` > `HKEY_LOCAL_MACHINE` > `SOFTWARE` > `Microsoft` > `Cryptography` > `MSCEP` > `EnforcePassword`.
+
+Change the value of `EnforcePassword` to `0` (default is `1`).
+
+Extend URL length for the request
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Best practices from Cisco recommand to extend the URL length to avoid issue with longer request. 
+
+To do so, enter the following command in the CLI on the NDES server:
+
+ %systemroot%\system32\inetsrv\appcmd.exe set config /section:system.webServer/
+ security/requestFiltering /requestLimits.maxQueryString:"16384" /commit:apphost
+
+Certificate template
+^^^^^^^^^^^^^^^^^^^^
+
+NOTE: Remember that the validity of your CA can impact your whole certificate architecture.
+
+The main goal is to deliver certificate for `user Authentication`, this mean we have to setup a specific template.
+
+First, the certificate template need at least the following `Enhanced Key Usage` and `Key Usage`:
+
+image::docs/images/scep-ms-pki-en-key-usage.png[scaledwidth="100%",alt="Mandatory Enhanced Key Usage"]
+
+image::docs/images/scep-ms-pki-key-usage.png[scaledwidth="100%",alt="Mandatory Key Usage"]
+
+Since we will duplicate a template, those `Key Usage`, and `Enhanced Key Usage` are already configured. We advise to duplicate the template `User` and change the necessary settings.
```

Comment:
Stop using 'we'. You are not doing it, the user is. Rephrase those so that it's impersonal